SK.API/appsettings.json

```diff
@@ -1,11 +1,15 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Host=localhost;Database=socialKnow;Username=postgres;Password=Start123"
+    "DefaultConnection": "Host=localhost;Database=socialKnow;Username=postgres;Password=1234"
```

Comment:
Create example file appsetings.json.simple with clear fields and descriptions. After this add this file to .gitignore.

Reply:
done